mono/cil/make-opcodes-def.pl

```diff
@@ -29,34 +29,35 @@
 
 sub handle_opcode {
     my ($parser, $elem, %attrs) = @_;
-    my ($name, $input, $output, $args, $o1, $o2, $flow, $uname, $count, $ff);
+    my ($count);
```

Comment:
The noise of useless changes in this file just drowns the signal of the changes which are actually useful.

Reply:
The changes are of varying value.
It is broadly acknowledged as better to initialize variables at point of declaration than to leave them uninitialized. So valuable that C99 gained that feature among little else.
It is easier for the human reader, post-diff, to follow the data/control flow, when the resulting data/control flow is smaller.
That trades off with the viewing of the diff to get to the better state.

Reply:
I understand the change, that's not the problem. I am just pointing out that the signal of what actually _needed_ to be fixed is drowned into the noise of what you _wanted_ to change.

Reply:
I agree with @luhenry's recommendation but on the other hand I feel like we can accept the current state as is without needing to go through another revision/build cycle :)